test/instrumentation/modules/aws-sdk/fixtures-dynamodb.js

```diff
@@ -0,0 +1,20 @@
+'use strict'
+module.exports = {
+  query: {
+    response: {
+      Count: 1,
+      Items:
+                [{ id: { S: '001' }, name: { S: 'Richard Roe' }, number: { N: '1' } }],
```

Comment:
nit: odd whitespace

Reply:
surprised lint:fix didn't do anything with this.  I've tightened up the indenting manually.